params/rules_extra.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func GetRulesExtra(r Rules) *RulesExtra {
-	return extras.PointerFromRules(&r)
+	return payloads.PointerFromRules(&r)
 }
 
 type RulesExtra struct {
```

Comment:
We could also move RulesExtra, but I don't see much advantage to doing it as it doesn't need to be externally serialized.

Reply:
Hehe I had to go back and delete an earlier comment asking about this. The rationale I'd given for moving it is that there's then a logical grouping of related functionality within the `extras` package. I'll leave it up to you as it's subjective.